devops/lib/component.py

```diff
@@ -76,9 +76,11 @@ def validate(self, ctx=None):
         if not ctx:
             return
 
+        skip_kinds = ",".join(KUBEVAL_SKIP_KINDS)
+
         for file in self.kube_configs:
             path = self.kube_configs[file]
-            result = run(["kubeval", path])
+            result = run(["kubeval", "--skip-kinds", skip_kinds, path])
```

Comment:
I imagine this also does not work well if `KUBEVAL_SKIP_KINDS = []`

Reply:
See the reply to the other question.